packages/babel-compat-data/data/overlapping-plugins.json

```diff
@@ -1,3 +1,18 @@
 {
-  "transform-regenerator": []
-}
\ No newline at end of file
+  "transform-async-to-generator": [
```

Comment:
Ideally yes. We did it in the past,but had to revert it because regenerator doesn't support async functions with polyfilled promises.

I'd like to revisit it in Babel 8 somehow.

Reply:
Okay. At least when bugfixes is enabled though, regenerator will never be included, right?

Reply:
It's only included when targeting browsers not supporting generators (we don't use it for async functions).

It's unrelated to `bugfixes`: it isn't included because of the `esmodules` option.